### DISCUSSION TITLE
Is it worthwhile to add a mention of transparent viewmodels disabling anti-aliasing to the docs quick-fixes section?

### DISCUSSION THREAD

**Discussion by aSeriousDeveloper** [mastercoms/tf2cfg#675] [Ideas]
Noticed I had this issue when I first installed it and helped someone out who seemed to have similar on the Discord.
Obviously, this is already documented within the app when choosing the addon, but it might be worthwhile to codify it in the docs also.

Additionally, it might also be worthwhile to add a section for addons to the customisation area of the docs to explain how/why the addons do what they do.

 can probably draft this up and add a PR to the repo if it's desirable.

**Comment by micwoj92** [comment]
prs welcome
